client/app/bundles/Attendance/containers/ClassContainer.jsx

```diff
@@ -10,8 +10,10 @@ import * as classesActionCreators from '../actions/classesActionCreators';
 function select(state) {
   return {
     data: {
+      belts: state.$$attendance.get("$$attendance").get("belts"),
       classes: state.$$attendance.get("$$attendance").get("classes"),
-      class_types: state.$$attendance.get("$$attendance").get("class_types"),
+      meeting_types: state.$$attendance.get("$$attendance").get("meeting_types"),
+      students: state.$$attendance.get("$$attendance").get("students"),
```

Comment:
This might be a dumb question, but why this `state.$$attendance.get("$$attendance")`?

Reply:
The variable name of the state is `$$attendance` and the name of the store is `$$attendance`. I'm not a huge fan of how it works, but I couldn't get around that.

Reply:
No big deal :)